src/main/java/se/sundsvall/supportmanagement/api/model/revision/DifferenceResponse.java

```diff
@@ -0,0 +1,60 @@
+package se.sundsvall.supportmanagement.api.model.revision;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+
+import java.util.List;
+import java.util.Objects;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "DifferenceResponse model")
+public class DifferenceResponse {
+
+	@ArraySchema(schema = @Schema(implementation = Operation.class, accessMode = READ_ONLY))
```

Comment:
Why not put `accessMode = READ_ONLY` in the class level `@Schema`-annotation.
It makes more sense, since all (existing and future) attributes in this modelclass most likely will be read-only.

I.e. on row 11:
`@Schema(description = "DifferenceResponse model", accessMode = READ_ONLY)`

Same goes for the Operation-class i suppose?